manifests/kustomize/base/pipeline-application.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
```

Comment:
Just confirmation:
* This is not GCP specific, is that correct?
* What value does this provide if not on GCP? Shall we consider putting it in an overlay?
* Should this be added to Kubeflow?

Reply:
this is not gcp specific but the goal of application CR is to provide a component to manage the monitor the application so i think this is still valuable for non gcp case. we should consider add to kubeflow manifest too.

Reply:
SG, I will take care of adding this to Kubeflow later.